icu4c/source/common/brkiter.cpp

```diff
@@ -448,7 +450,7 @@ BreakIterator::makeInstance(const Locale& loc, int32_t kind, UErrorCode& status)
             }
             result = BreakIterator::buildInstance(loc, lbType, status);
 
-            UTRACE_DATA1(UTRACE_INFO, "lb=%s", value.data());
+            UTRACE_DATA1(UTRACE_INFO, "lb=%s", lbType);
```

Comment:
now the tracing output is not really anymore but $lb[_$lw] right? should we change it to
```
UTRACE_DATA1(UTRACE_INFO, "lb_lw=%s", lb_lw);
```
instead?